metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/UserMetadataService.java

```diff
@@ -84,7 +92,9 @@
      * @return data metadata for the given uri.
      */
     @Nonnull
```

Comment:
This makes no sense. The entire purpose of returning an Optional is so that you can check if the contents are not null?

Are we just saying the actual Optional itself is nonnull? that's weird.